src/actix/api/local_shard_api.rs

```diff
@@ -153,78 +139,18 @@ async fn count_points(
     .await
 }
 
-#[post("/collections/{collection}/shards/{shard}/points/delete")]
-async fn delete_points(
+#[post("/collections/{collection}/shards/{shard}/cleanup")]
```

Comment:
Since we changed this and don't strictly take a body anymore, is POST still a good candidate here? :thinking:

Reply:
Yeah, I think it is. It's definitely not `GET`, and `PUT`/`PATCH`/`DELETE` does not feel better as well.